### DISCUSSION TITLE
Share your ideas that can be added

### DISCUSSION THREAD

**Discussion by MRMYSTERY003** [MRMYSTERY003/sketchpy#16] [Ideas]
It can be anything!

**Comment by ashutoshdebug** [comment]
**I am getting the following error, can you help in this?**

![image](https://github.com/MRMYSTERY003/sketchpy/assets/126601693/5c3547bb-1127-4353-80a9-621f6249d758)

**Reply by MRMYSTERY003** [reply]
let me check, and i will inform you once it is finished, until then you can use the older version of sketchpy
